pkg/cmd/cli/cmd/edit.go

```diff
@@ -211,7 +211,11 @@ func RunEdit(fullName string, f *clientcmd.Factory, out io.Writer, cmd *cobra.Co
 			if err != nil {
 				return err
 			}
-			updated, err := resource.NewHelper(info.Client, info.Mapping).Update(info.Namespace, info.Name, false, data)
+			namespace, err := info.ResourceMapping().Namespace(obj)
```

Comment:
If you mean `cmdNamespace` it's available here, but not in `applyPatch` and `AddError` (unless I pass it).